_inc/lib/markdown/gfm.php

```diff
@@ -60,7 +60,16 @@ class WPCom_GHF_Markdown_Parser extends MarkdownExtra_Parser {
 	 */
 	public function __construct() {
 		$this->use_code_shortcode  = class_exists( 'SyntaxHighlighter' );
-		$this->preserve_shortcodes = function_exists( 'get_shortcode_regex' );
+		/**
+		 * Allow processing shortcode contents
```

Comment:
Could you add a period at the end of this description, as well as at the end of the `@param` declaration, to avoid errors for those of us using a code sniffer?

Thanks!